packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -51,6 +53,13 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
 
   const emailsToInvite = await getEmailsToInvite(input.usernameOrEmail);
 
+  if (emailsToInvite.length > 25) {
+    throw new TRPCError({
+      code: "PAYLOAD_TOO_LARGE",
+      message: "Please invite 25 members at a time",
```

Comment:
```suggestion
      message: "Please invite 50 members at a time",
```

Reply:
lets do 50 instead

Reply:
also, is there no better way for us to rate limit this? 🤔

Reply:
IMO this is uglier than a global "You can only have 50 pending invites" - please poke your team members to send more.

with an exception of verified orgs

Reply:
agree. lets just make invites paid. that will solve the spam issues. closing